client/src/components/ElementEditor/Header.js

```diff
@@ -111,6 +124,31 @@ class Header extends Component {
     );
   }
 
+  /**
+   * Render the unpublish button if the current state of the button is published
+   *
+   * @returns {DOMElement|null}
+   */
+  renderUnpublishButton() {
+    const { isPublished } = this.props;
+    const unpublishTitle = i18n._t('ElementHeader.UNPUBLISH', 'Unpublish');
+
+    if (!isPublished) {
```

Comment:
This prop has been renamed to isLiveVersion - does this need updating too?

Reply:
No, I don't think so. We won't to always give the CMS user the option to unpublish the publish version. By using `isPublished` here instead of `isLiveVersion` we ensure that they can also unpublish in the case that a version has been published and then modified without being published. In this case `isPublished` equals true, whereas `isLiveVersion` equals false.